src/oscar/apps/voucher/migrations/0004_auto_20170417_0831.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-04-17 07:31
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def move_date_created_to_datetime_field(apps, schema_editor):
+    Voucher = apps.get_model('voucher', 'Voucher')
+    VoucherApplication = apps.get_model('voucher', 'VoucherApplication')
+    for voucher in Voucher._default_manager.all():
+        voucher.date_created = voucher.date_created_original
+        voucher.save()
```

Comment:
Besides my earlier remark, this should always be done via a simple update statement:

```python
Voucher.objects.update(date_created=F('date_created_original'))
VoucherApplication.objects.update(date_created=F('date_created_original'))
```
But I think we really should just alter the type of the field with one single migration instead.